media/src/main/java/edu/artic/media/audio/AudioPlayerService.kt

```diff
@@ -258,19 +278,38 @@ class AudioPlayerService : DaggerService(), PlayerService {
 
 
                     override fun getCurrentLargeIcon(player: Player?, callback: PlayerNotificationManager.BitmapCallback): Bitmap? {
-                        playable?.getPlayableThumbnailUrl()?.let {
-
-                            Glide.with(this@AudioPlayerService)
-                                    .asBitmap()
-                                    .load(it)
-                                    .into(BitmapCallbackTarget(callback))
+                        if (currentBitmap == null) {
+                            playable?.getPlayableThumbnailUrl()?.let {
+                                Glide.with(this@AudioPlayerService)
+                                        .asBitmap()
+                                        .load(it)
+                                        .into(BitmapCallbackTarget(this@AudioPlayerService, callback))
+                            }
                         }
-                        return null
+                        return currentBitmap
+                    }
+                },
+                object : PlayerNotificationManager.CustomActionReceiver {
+                    override fun getCustomActions(player: Player?): MutableList<String> {
+                        return mutableListOf("Cancel_Notification")
```

Comment:
Please create a `val` in the CustomActionReceiver object for `"Cancel_Notification"`.